neuralforecast/common/_modules.py

```diff
@@ -398,6 +398,59 @@ def forward(self, x):
         return self.embed(x)
 
 
+class FixedEmbedding(nn.Module):
+    def __init__(self, c_in, d_model):
+        super(FixedEmbedding, self).__init__()
+
+        w = torch.zeros(c_in, d_model).float()
+        w.require_grad = False
```

Comment:
I'd use torch.zeros(..., dtype=torch.float32, requires_grad=False), which (i) avoids a type conversion and (ii) removes necessity of the next line.
```suggestion
        w = torch.zeros((c_in, d_model), dtype=torch.float32, requires_grad=False)
```

Reply:
done